EscapeHost/src/main/java/nl/tudelft/kroket/server/Settings.java

```diff
@@ -25,4 +25,6 @@
 
   /** Number of threads available in the pool. */
   public static final int THREAD_POOL_MAX = 10;
+
+  public static final int TIMELIMIT = 23 * 60;
```

Comment:
I used an odd value to test this, as the VR client has a different default value. This value will overrule the timer in the VR client.